src/System.Windows.Forms/src/System/Windows/Forms/ToolStripItem.cs

```diff
@@ -481,19 +624,36 @@ public virtual Color BackColor
             }
         }
 
-        [SRCategory(nameof(SR.CatPropertyChanged))]
-        [SRDescription(nameof(SR.ToolStripItemOnBackColorChangedDescr))]
-        public event EventHandler BackColorChanged
-        {
-            add => Events.AddHandler(s_backColorChangedEvent, value);
-            remove => Events.RemoveHandler(s_backColorChangedEvent, value);
-        }
+        /// <summary>
+        /// The bounds of the item
+        /// </summary>
+        [Browsable(false)]
+        public virtual Rectangle Bounds
+            => _bounds;
```

Comment:
We generally don't wrap short lines.
```suggestion
        public virtual Rectangle Bounds => _bounds;
```

Ditto other places